bootloader/wscript

```diff
@@ -661,6 +661,9 @@ def build(ctx):
     if ctx.env.CC_NAME == 'msvc':
         # Do not strip bootloaders when using MSVC.
         features = ''
+        
+        ctx.env['WINRC_TGT_F'] = '/fo'
+        ctx.env['WINRC_SRC_F'] = ''
```

Comment:
Here is a [clean build][1] to show that it really does work on x64. Really I don't think too much time should be spent on this if two lines in configure can fix the issue.

[1]: https://ci.appveyor.com/project/htgoebel/pyinstaller/build/1.0.163